docs/ref/contrib/admin/index.txt

```diff
@@ -714,6 +714,21 @@ subclass::
 
           colored_first_name.admin_order_field = '-first_name'
 
+      ``admin_order_field`` supports :ref:`field lookups <field-lookups>`.
```

Comment:
field lookups doesn't seem to be the correct term, that's things like __gt __contains, etc. -- I would say, "``admin_order_field`` supports query lookups to sort by values of fields on related models::"